sbt-plugin-test/referencedCrossProject/build.sbt

```diff
@@ -1,7 +1,7 @@
 lazy val referencedCrossProject = crossProject.
   crossType(CrossType.Pure).
   in(file(".")).
-  settings(scalaVersion := "2.11.7")
+  settings(scalaVersion := "2.11.8")
```

Comment:
Maybe this helps for the future: http://ant.apache.org/ivy/history/latest-milestone/tutorial/defaultconf.html

ivy.local.default.root=/local/path/to/ci/run

Just don't know where to put this guy